src/test/java/org/openmrs/contrib/qaframework/automation/RegistrationSteps.java

```diff
@@ -12,21 +11,23 @@
 import org.openmrs.reference.page.RegistrationPage;
 import org.openqa.selenium.By;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import io.cucumber.java.After;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
 
 public class RegistrationSteps extends Steps {
 
-	RegistrationPage registrationPage;
-	TestPatient patient;
-	ClinicianFacingPatientDashboardPage dashboardPage;
+	private RegistrationPage registrationPage;
+	private TestPatient patient;
+	private ClinicianFacingPatientDashboardPage dashboardPage;
```

Comment:
@sherrif10 make use of inheritance and get rid of `ClinicianFacingPatientDashboardPage`

Reply:
> @sherrif10 make use of inheritance and get rid of `ClinicianFacingPatientDashboardPage`

ohh where would I use inheritance here, Do you mind suggest an example.

Reply:
`RegistrationSteps` is a child class of `Steps` and  by default variables that are declared within the parent class can be used in a child class.

Reply:
> `RegistrationSteps` is a child class of `Steps` and by default variables that are declared within the parent class can be used in a child class.

is this really supposed to be implemented in this class workflow ? i think you are talking about something else

Reply:
@kdaud  any more reviews.

Reply:
Variables that are declared in a parent class can directly be used by a child class, so whenever a variable declared in a parent class you do not need to declare it in a child class to eliminate code redundancy !

@sherrif10 do you think you need again to declare this variable [here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/d5f431cbc90972e1a5a6eca0b955f7edc26ee35d/src/test/java/org/openmrs/contrib/qaframework/automation/RegistrationSteps.java#L23) ?

Reply:
Resolved thanks.